utils/networks.ts

```diff
@@ -9,6 +11,9 @@ export type NetworkConfig = {
   railgunNetworkName: NetworkName;
   chainId: number;
   wethAddress: string;
+  evmGasType: EVMGasType;
+  baseToken: { symbol: string; name: string };
+  fallbackProviders: FallbackProviderJsonConfig;
```

Comment:
(arbitrary place for a thread)

Noticing this error on polygon when I try to shield. And also should the native token be MATIC rather than Polygon. I wonder if that is an SDK issue

![Screenshot from 2023-01-20 11-01-15](https://user-images.githubusercontent.com/12705146/213746812-89d83326-fd6c-4383-b503-8afc122fc54e.png)

Reply:
thanks, we've got the base token names in our network object, changed to MATIC: 2220e8e

will dig into the polygon error

Reply:
We need to enforce a 30gwei min priority fee in polygon, which is required by the network. Looks like we're setting a 1.5gwei max priority fee 
<img width="208" alt="image" src="https://user-images.githubusercontent.com/17163988/213808028-2c802c4a-bb91-4eed-8773-4aa3fe5d28b6.png">

Reply:
- default `PriceLimit` is 30 gwei: https://github.com/maticnetwork/bor/blob/d6899d79ea4c4d169d9dde613a9f4b0f1dccc3d8/builder/files/config.toml#L44
- the `validateTx` method returns tx underpriced error when `tx.gasTipCapU256` is less than that value https://github.com/maticnetwork/bor/blob/d6899d79ea4c4d169d9dde613a9f4b0f1dccc3d8/core/tx_pool.go#L769-L772

Reply:
e1ce4da 30 gwei min max enforced